README.md

```diff
@@ -1535,6 +1539,65 @@ For convenience, this has been aliased to `.spa`:
 await stringSchema.spa("billie");
 ```
 
+### `.check`
```

Comment:
Yeah, I think maybe we can iterate on the name to make it more obvious what it does, let me spend a few minutes spitballing and brainstorming:

- `parseInput`/`safeParseInput`: We're still parsing rather than validating, so maybe this makes that more clear and makes it more obvious that the argument it expects is of type `Input`?
- `decode`/`safeDecode`: Maybe this is closer to a codec kind of thing where you have some defined input type that you're decoding from?
- `construct`/`safeConstruct`: Maybe these are "constructors" of this type? Maybe too much of an overloaded term?
- `make`/`safeMake`: Taking a page from the OCaml/ReScript world, most constructors there are named `make`.

Reply:
I like decode, I feel like it's the name that describes best the fact that transformations are applied?